gp_emulator/GaussianProcess.py

```diff
@@ -283,13 +368,14 @@ def hessian ( self, testing ):
     #target2 = data[ :,1]
     #target3 = data[ :,2]
     #input_obs = data[ :, 3: ]
-    wheat = np.loadtxt("argentine_wheat.dat")[:, 1:]
```

Comment:
Avoid hardcoded paths like the plague. Use python argparse library to specify input files.